README.md

```diff
@@ -1,15 +1,34 @@
 # Lokalise - the unofficial node lokali.se client.
 
-This is all still very WIP.
-
-## How to use
+### Configuration
 
 In your working directory create a `.lokalise.json` file with the following contents:
 
 ```
 {
-  "api_token": "<YOUR LOKALI.SE API TOKEN>",
-  "project_id": "<YOUR PROJECT ID>",
-  "output_path": "<PATH WHERE LOCALIZATION FILES WILL BE PUT>"
+  "api_token": [string] (required) Your locali.se api token,
+  "project_id": [string] (required) Your locali.se project ID,
+  "output_path": [string (required) Path where messages files will be downloaded,
+  "keys_file": [boolean | object] (optional, default false) Creates a keys file under output_path (more below)
+}
+```
+
+The optional `keys_file` configuration creates a file under `output_path` that enumerates your locali.se message keys.
+
+This may be very useful in your code, for autocompletion & type checking over the allowable message keys, for example.
+
+You can configure the format of this keys file to suit your needs by passing the following object:
+
+```
+{
+  name: [string] (optional) A name for the keys file, defaults to keys.{type-appropriate-file-extension},
+  type: 'json' | 'es6_module' | 'es5_module' (optional, default 'json') the type of the output keys file
+  flow: [boolean] (optional, default false) in the case {type} is a js module, adds a //@flow annotation at the top so that the keys can be used with flow type checking
 }
 ```
+
+Alternatively, passing `"keys_file": true` gives you the all default config - i.e. a `keys.json` file will be output
+
+### Run
+
+`yarn lokalise` or `npm run lokalise`
```

Comment:
Erm, there's no such "script" in the npm package :D. It's a executable, I don't think we need documentation on how to run it in the readme. Usage instructions should be generated by commander, we could add those once I brush them up.

Reply:
Ah ok - I didn't actually try `npm run` but `yarn` (which I am using) will fall through to running the binary for the package if there is no script present in packages.json, which is a pretty nice feature (basically you don't have to create one of those dummy `"package": "package"` scripts). I'll take out the npm one.

Reply:
Well, if you link it as a global executable via `npm install -g lokalise` or `yarn global install lokalise`, then you can use it wherever you like outside the context of package managers.